tests/unit/type/list.tcl

```diff
@@ -325,6 +511,36 @@ start_server {
         r lrange blist 0 -1
     } {a b c}
 
+    test "BRPOPRPUSH maintains order of elements after failure" {
```

Comment:
i think we can trim these 3 tests, they don't add value.

Reply:
Done.